core/tests/puppeteer-acceptance-tests/blog-admin-tests/blog-admin-assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,39 @@
+const e2eSuperAdmin = require('../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const testConstants = require('../puppeteer-testing-utilities/testConstants.js');
+
+const blogAdminUrl = testConstants.URLs.BlogAdmin;
+
+async function blogAdminUpdatingRolesAndTagsProperties() {
+  const superAdmin = await new e2eSuperAdmin();
+
+  await superAdmin.openBrowser();
+  await superAdmin.signUpNewUserWithUserNameAndEmail('blogAdm', 'blog_admin@example.com');
+  await superAdmin.logout();
+
+  await superAdmin.signUpNewUserWithUserNameAndEmail('blogPostEditor', 'blog_post_editor@example.com');
+  await superAdmin.logout();
+
+  await superAdmin.signUpNewUserWithUserNameAndEmail('superAdm', 'testadmin@example.com');
+  await superAdmin.assignBlogAdminRoleToUserWithUserName('superAdm');
+  await superAdmin.expectUserToHaveBlogAdminRole();
```

Comment:
Shouldn't you be passing in a username for this? Think carefully about what the contracts of the methods should be.